src/components/StatsBoxGrid.js

```diff
@@ -105,8 +103,50 @@ const LoadingMessage = () => (
   </IndicatorSpan>
 )
 
+const Lines = styled.div`
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 65%;
+`
```

Comment:
![Screenshot 2021-09-24 at 11 05 39](https://user-images.githubusercontent.com/62268199/134657475-905de759-9955-44d2-93c7-212dfbe50cec.png)

Currently the `Lines` cut through the `Value` on a dip. We could add some bottom here and reduce the height to mitigate this a bit more.

![Screenshot 2021-09-24 at 11 05 04](https://user-images.githubusercontent.com/62268199/134657868-ad391ff9-4ba0-437d-93ff-ae87370722ab.png)